examples/scripts/ppo-sentiment-t5-small.py

```diff
@@ -15,17 +15,16 @@
 import torch
 from tqdm import tqdm
 tqdm.pandas()
-
-from transformers import pipeline, AutoTokenizer
+from transformers import pipeline, AutoTokenizer, AutoModelForSeq2SeqLM
 from datasets import load_dataset
 
 from trl import PPOTrainer, PPOConfig, AutoModelForSeq2SeqLMWithValueHead
-from trl.core import LengthSampler
+from trl.core import LengthSampler, logprobs_from_logits
 
 ########################################################################
 # This is a fully working simple example to use trl with accelerate.
 #
-# This example fine-tunes a T5 model on the IMDB dataset using PPO 
+# This example fine-tunes a GPT2 model on the IMDB dataset using PPO 
```

Comment:
that was a fix too :)

Reply:
Addressed in [d0875b5](https://github.com/lvwerra/trl/pull/93/commits/d0875b5eab233993493d6390be2d2bfe5d2e707a)